### PR TITLE
[coverage] Scapy version

### PR DESCRIPTION
**Pull request by guedou**
This PR extends the coverage of `scapy/__init__.py`. it also provides a better naming scheme when using git archive.

### CODE REVIEW

File: test/regression.uts [hunk 1]
```diff
@@ -8928,4 +8928,21 @@ if WINDOWS:
...
+= _version()
+
+version = scapy._version()
+import os
+assert(os.path.exists("scapy/VERSION"))
+
+import mock
+with mock.patch("scapy._version_from_git_describe") as version_mocked:
+  version_mocked.side_effect = Exception()
+  assert(scapy._version() == "2.3.3.dev696")
```
Comment (gpotter2) [inline test/regression.uts]: Isn't this going to change ?

Comment (ThomasFaivre) [reply test/regression.uts]: I don't know how the mock library works, but it looks like it has been integrated as a standard library in python 3 in the unittest module. So I'm not sure 'import mock' will work

Comment (gpotter2) [reply test/regression.uts]: Nah this point works :) We're manually installing the mock module as unittest.mock isn't available on python 2.7.

**Comment by guedou** [comment]
@p-l- ready to be reviewed.

**Pull request merged**